src/test/java/com/softserveinc/ita/pageobjects_task/nromanchuk/GoogleTest.java

```diff
@@ -1,14 +0,0 @@
-package com.softserveinc.ita.pageobjects_task.nromanchuk;
```

Comment:
DO NOT CHANGE someone else's code

Reply:
I accidentally did it and did not notice. I will be more attentive

Reply:
I accidentally did it and did not catch sight of. I will be more attentive

Reply:
revert